`variables/options` should only be required if `TVariables` is empty or default (#11241)
apollographql/apollo-client: .changeset/slow-planets-design.md

---
"@apollo/client": patch
---

> useQuery: `variables/options` should only be required if `TVariables` is empty or default

Author: ```suggestion
useQuery: `variables/options` should be required if `TVariables` is not empty/purely optional/default
```
